restapi/src/test/java/io/stargate/web/resources/WhereParserTest.java

```diff
@@ -288,4 +308,24 @@ public void testInOperation() throws IOException {
 
     assertThat(where).isEqualTo(whereExpected);
   }
+
+  @Test
+  public void testInWithTimestampOperation() throws IOException {
+    String whereParam = "{\"created\":{\"$in\":[\"2021-04-23T18:42:22.139Z\"]}}";
+    List<BuiltCondition> whereExpected =
+        singletonList(
+            BuiltCondition.of(
+                "created", Predicate.IN, singletonList(Instant.parse("2021-04-23T18:42:22.139Z"))));
```

Comment:
nit: maybe use `Converters.toCqlValue(type, object)` instead of directly calling `Instant`?